lib/DDG/Spice/JsKeycodes.pm

```diff
@@ -0,0 +1,39 @@
+package DDG::Spice::JsKeycodes;
+# ABSTRACT: Get the Javascript keycode of a key
+
+use DDG::Spice;
+use Text::Trim;
+
+spice is_cached => 1;
+
+# Metadata
+name "Javascript Keycodes";
+source "MDN";
+description "Get the Javascript keycode for a key";
+primary_example_queries "keycode j", "javascript keycode tab";
+secondary_example_queries "javascript keycodes";
+
+category "reference";
+topics "programming";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/JsKeycodes.pm";
+attribution github => ["https://github.com/andrey-p/", "Andrey Pissantchev"];
+
+# Triggers
+triggers any => "js keycode",
+    "javascript keycode",
+    "js keycodes",
+    "javascript keycodes",
+    "js key code",
+    "javascript key code",
+    "js key codes",
+    "javascript key codes";
+
+
+spice call_type => 'self';
+
+# Handle statement
+handle remainder_lc => sub {
+    return call;
```

Comment:
We should check here that if there's a remainder it's relevant. I.e. just a single word -- if we could make sure it's exactly an expected keycode that would be awesome. We could build a hash (perhaps in a separate text file in /share/) that contains all the names of the keys and if the remainder is one of those, or non-existant this AI can return, otherwise it should ignore the query.

We typically use YAML and load the strings into a hash for a fast lookup. This is a good example: https://github.com/duckduckgo/zeroclickinfo-spice/pull/1737/files